assets/js/main.js

```diff
@@ -1,8 +1,8 @@
 $('body').on({
-  mouseenter: function() {
+  mouseenter: function () {
     var submenu = $(this).find(".sub-menu");
 
-    if(!$(this).hasClass('submenu-item')) {
+    if (!$(this).hasClass('submenu-item')) {
```

Comment:
Could you avoid the change in spacing please 🙏  or add this in another commit.  Otherwise when we look for a change in this functionality somewhere in the future, this change will be misleading. There is no real change but tabulation.

Reply:
If I'm not being clear you can ask me directly.

Reply:
ok, no problem